src/snowflake/snowpark/dataframe.py

```diff
@@ -2677,6 +2677,8 @@ def convert(col: ColumnOrName) -> Expression:
     withColumnRenamed = with_column_renamed
     toLocalIterator = to_local_iterator
     randomSplit = random_split
+    order_by = sort
```

Comment:
this alias is not necessary for migration purpose.

Reply:
**ORDER BY** is part of [standard SQL clause](https://en.m.wikipedia.org/wiki/Order_by), also included in [Apache-Spark](https://spark.apache.org/docs/3.1.1/api/python/reference/api/pyspark.sql.DataFrame.orderBy.html)

Reply:
Yep, I mean we will definitely add `orderBy` here for migration purpose, but I'm not sure whether `order_by` is needed, because it's not usual in dataframe API.

Reply:
We can keep order_by because we want to have a snake_case API.